public_html/index.html

```diff
@@ -52,44 +52,55 @@
 	  </div>
       </nav>
       <article class="well">
-	
-	<img class="img-responsive" src="images/road.jpg" alt="picture of road">
-
-	<h1>
-	  This new product will solve all the world’s problems
-	</h1>
-
-	<p>
-	  Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed 
-	  diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam
-	  erat, sed diam voluptua. At vero eos et accusam et justo duo dolores
-	  et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est
-	  Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur
-	  sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore
-	  et dolore magna aliquyam.
-	</p>
-
-	<a class="btn btn-warning btn-block btn-lg" href="#">Sign Up</a>
+	<div class="row">
+	  <div class="col-sm-6">
+	    <img class="img-responsive" src="images/road.jpg" alt="picture of road">
+	  </div>
+	  <div class="col-sm-6">
+	    <h1>
+	      This new product will solve all the world’s problems
+	    </h1>
 
+	    <p>
+	      Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed 
+	      diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam
+	      erat, sed diam voluptua. At vero eos et accusam et justo duo dolores
+	      et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est
+	      Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur
+	      sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore
+	      et dolore magna aliquyam.
+	    </p>
+	  </div>
+	</div>
+	<div class="row">
+	  <div class="col-sm-8"></div>
```

Comment:
Instead of this empty div have a read of the Bootstrap documentation on the grid system. There should be a more elegant way to offset the button